.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
```

Comment:
- [ Optional ] good job for adding a link to the home page in the logo but it would be better to add a separate link in the navbar  even if it is not mentioned in the design the user doesn't have to guess where he can go to the home page from 
![image](https://user-images.githubusercontent.com/74744787/211168186-2f8be581-0edd-4643-bcb4-ed8d514f99df.png)

- [Optional] I just one to add my vote to the last reviewer the optional suggestions he mentioned are so important : 
- add a black part to the about page navbar

Reply:
@MeqdamAlqudah 
Do you mean the small home button? That should fix the issue of the user having to guess how to go to the home page. Will fix that. Thanks for your suggestions.